### ISSUE TITLE
Redirect on api error

### ISSUE THREAD

**Issue by 4ware** [benawad/react-menu-monkey-client#6]
Hi!

When receiving an error from the api, like when the login is rejected, you just return {}, which isn't very helpfull.

How would you do it so that one could display an error to the client and rerender the form?

Cheers, Kai

**Comment by benawad** [comment]
One possible option, is to put any errors in the redux state. Then in your form component, you can check if there are any errors and display a message

**Comment by 4ware** [comment]
Would you mind to update the code for this? I don't know how... :(

**Comment by benawad** [comment]
Sure. I'll code it up tomorrow

**Comment by benawad** [comment]
I coded it in a separate branch: https://github.com/benawad/react-menu-monkey-client/tree/handle-login-error

I added error handling for the login page. So if a user tries to login with an invalid email or password it will stop them. I recommend taking a look at this commit: https://github.com/benawad/react-menu-monkey-client/commit/c8cab6dcdee333d8a4f2af0bd28f2824fb0bd454

It shows exactly what I changed.
